src/Pages/Explorar/Explorar.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import Header from '../components/Header';
+import Header from '../../components/Header';
 
-import MenuInferior from '../components/MenuInferior';
+import MenuInferior from '../../components/MenuInferior';
 
 function Explorar() {
   return (
     <div>
       <Header />
-      <Link to="/comidas">
+      <Link to="/explorarcomidas">
```

Comment:
Fala Diego blz? Cara eu reparei que no README do projeto a rota está desta forma _/explorar/comidas_ o que acha de alterar a rota aqui e no `<app />`?

```suggestion
      <Link to="/explorar/comidas">
```